MFVNeutralino/python/NtupleCommon.py

```diff
@@ -346,14 +355,37 @@ def ntuple_process(settings):
     else:
         return aod_ntuple_process(settings)
 
+def signal_uses_random_pars_modifier(sample): # Used for samples stored in inclusive miniaods
+    to_replace = []
+
+    if sample.is_signal:
+        if sample.name.startswith('ZH_') or sample.name.startswith('Wplus'):
+            magic_randpar = 'randpars_filter = False'
+            
+            decay = sample.name[sample.name.find('_')+1 : sample.name.find('_Z')]
```

Comment:
This is missing Wminus samples. And does line 365 work correctly for WH? The "_Z" makes me think it would not. Do we have any other randpar samples currently? @pkotamnives @abbywarden